install_linux.sh

```diff
@@ -50,6 +55,13 @@ if [[ -z "$db_pw" ]] ; then
     read -sp "Please enter the new password to be set for the PostgreSQL DB: " db_pw
 fi
 
+if [[ $ask_tmpfs == true ]] ; then
+    read -p "We strongly recommend mounting /tmp on a tmpfs. Do you want to do that? (y/N)" tmpfs
+    if [[ "$tmpfs" == "Y" || "$tmpfs" == "y" ]] ; then
+        sudo systemctl enable /usr/share/systemd/tmp.mount
```

Comment:
We need a check there for fedora. The command under fedora is
```
sudo systemctl unmask --now tmp.mount
```

Just enabling tmp.mount does not work as it is not called by itself but wanted by local-fs.target

Reply:
cool, ty!